docs/source/production.rst

```diff
@@ -106,6 +106,18 @@ SQLFluff comes with two `pre-commit`_ hooks:
 * sqlfluff-lint: returns linting errors.
 * sqlfluff-fix: attempts to fix rule violations.
 
+.. note::
+   For safety reasons, ``sqlfluff-fix`` by default will not make any fixes in
+   files that had parse errors.
+
+   Although it is not advised, you _can_ tell SQLFluff to try and fix
```

Comment:
What does the underline mean? Should it be in italics?

```suggestion
   Although it is not advised, you `can` tell SQLFluff to try and fix
```

Reply:
Oh yeah. Markdown confusion. Will fix.